### PR TITLE
Conversion of Json document in to native format - part 1 intermediate

### PR DESCRIPTION
**Pull request by SeanCondon**

### CODE REVIEW

File: pkg/store/json2Values.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
+// Copyright 2019-present Open Networking Foundation.
```
Comment (Andrea-Campanella) [inline pkg/store/json2Values.go]: I know I might be old fashioned, but I'd go for `jsonToValues.go`

File: pkg/store/json2Values.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+				changes = append(changes, o)
+			}
+		}
+	case string:
+		newCv := change.ConfigValue{Path: parentPath, TypedValue: *change.CreateTypedValueString(value)}
+		changes = append(changes, &newCv)
+	case float64:
+		newCv := change.ConfigValue{Path: parentPath, TypedValue: *change.CreateTypedValueFloat(float32(value))}
+		changes = append(changes, &newCv)
+	default:
```
Comment (Andrea-Campanella) [inline pkg/store/json2Values.go]: is there a boolean type ?

File: pkg/store/json2Values.go (hunk 1 above)
Comment (SeanCondon) [reply pkg/store/json2Values.go]: done

File: pkg/store/json2Values.go (hunk 2 above)
Comment (SeanCondon) [reply pkg/store/json2Values.go]: added - good catch

**Pull request merged**